src/js/components/RulePicker.react.js

```diff
@@ -87,10 +88,11 @@ class RulePicker extends React.Component<Props, State> {
               valid: this.props.rule.selector != '',
             })}
           >
-            <label>
-              {this.props.rule.selector != '' ? <span>✔</span> : <span>✘</span>}{' '}
-              Selector
-            </label>
+            {this.props.rule.selector != '' ? (
+              <RuleLabel level="3" title="Selector" />
+            ) : (
+              <RuleLabel level="2" title="Selector" />
+            )}
```

Comment:
After migrating to 2 boolean properties, let's refactor all conditions like this to simply:

```js
<RuleLabel filled={this.props.rule.selector != ''} required={true} title="Selector" />
```